### ISSUE TITLE
Node version error when linking worspace

### ISSUE THREAD

**Issue by igorpoubel** [vtex-apps/store-discussion#692]
**Describe the bug**
Suddenly, after everything link well, the link started to show this error. Saying that the node engine is incompatible with react-hook-form module. This is the secound time this happens. The first time we rebuilt everything guessing it was soma other package interfearing. But I think it's better if this get resolved. To never happens again, or at least we know what to do.

**To Reproduce**
I don't know how to reproduces. Because it happens without precedent.

**Expected behavior**
Link the project without errors.

**Screenshots**
![image](https://user-images.githubusercontent.com/6241622/181630892-48f29d1b-07a6-4b6f-8632-9416ccb6dcaa.png)
Error

![image](https://user-images.githubusercontent.com/6241622/181631148-b905f121-13ae-4032-a707-8d2e4246c2ef.png)
vtex --version and node verssion

**Desktop environment (please complete the following information, if applicable):**

![image](https://user-images.githubusercontent.com/6241622/181631352-6c534fee-15ae-4c1a-848c-fd636ed964c7.png)

**Additional context**
I don't think it's this package (react-hook-form), becausa it is used at least almost a year without erros besides that one I told.

**Comment by leandropfs** [comment]
Same here

**Comment by nicholasessacct** [comment]
@igorpoubel @leandropfs Where I work, the team solve the problem installing exactly the version 6.9.4

**Comment by igorpoubel** [comment]
@nicholasessacct I forgot to report here. I did that some days ago and solved my "problem". Talking to some people from Vtex, they said that vtex.builder-hub has 12.16.1 node version. And, on React-Hook-Form, the node version required is 12.22. That's the reasons of this error. So, looking at RHF repository on GitHub, I found the version that node requirements is lower than, and leave RHF in the exactly version.

@leandropfs try to do that. Probably will solve your problem too.
